cumulusci/cli/tests/test_cci.py

```diff
@@ -1813,55 +1792,10 @@ def test_task_info(self, doc_task, rst2ansi):
         doc_task.assert_called_once()
         rst2ansi.assert_called_once()
 
-    def test_task_run(self):
-        runtime = mock.Mock()
-        runtime.get_org.return_value = (None, None)
-        runtime.project_config = BaseProjectConfig(
-            None,
-            config={
-                "tasks": {
-                    "test": {"class_path": "cumulusci.cli.tests.test_cci.DummyTask"}
-                }
-            },
-        )
-        DummyTask._run_task = mock.Mock()
-
-        run_click_command(
-            cci.task_run,
-            runtime=runtime,
-            task_name="test",
-            org=None,
-            o=[("color", "blue")],
-            debug=False,
-            debug_before=False,
-            debug_after=False,
-            no_prompt=True,
-        )
-
-        DummyTask._run_task.assert_called_once()
-
-    def test_task_run_invalid_option(self):
-        runtime = mock.Mock()
-        runtime.get_org.return_value = (None, None)
-        runtime.project_config.get_task.return_value = TaskConfig(
-            {"class_path": "cumulusci.cli.tests.test_cci.DummyTask"}
-        )
-
-        with self.assertRaises(click.UsageError):
-            run_click_command(
-                cci.task_run,
-                runtime=runtime,
-                task_name="test",
-                org=None,
-                o=[("bogus", "blue")],
-                debug=False,
-                debug_before=False,
-                debug_after=False,
-                no_prompt=True,
-            )
-
     @mock.patch("pdb.set_trace")
     def test_task_run_debug_before(self, set_trace):
```

Comment:
It looks like you re-created some of these in test_run_task.py and they can probably be removed from here?